activemodel/lib/active_model/validations/callbacks.rb

```diff
@@ -23,6 +23,7 @@ module Callbacks
       included do
         include ActiveSupport::Callbacks
         define_callbacks :validation,
+                         terminator: deprecated_false_terminator(ActiveModel::Validations),
```

Comment:
Ok… maybe the idea of passing the module was not great.

I just thought of having something to pass through, so it would make for a better deprecation error.

If we just said "returning false in callbacks is deprecated", that would be misleading, because it's not in *all* callbacks, just AR and AM ones.

@kaspth… given all your feedback, I wonder whether we should drop the idea of passing these modules around at all, and simply fix the string in the `deprecated_false_terminator` model to say "returning false in ActiveRecord and ActiveModel callbacks is deprecated" and call it a day.